client_dashboard/common/src/commonMain/kotlin/org/thechance/common/domain/entity/Restaurant.kt

```diff
@@ -0,0 +1,11 @@
+package org.thechance.common.domain.entity
+
+data class Restaurant(
+    val id: String,
+    val restaurantName: String,
```

Comment:
```suggestion
    val name: String,
```
no need to mention the restaurant name in the variable the data class already mentions it